common/scripted_effects/wc_artifacts_gamestart_scripted_effects.txt

```diff
@@ -0,0 +1,335 @@
+game_start_spawn_artifact_amulet_1_effect = {
+	random_character = {
+		limit = { can_get_amulets_at_gamestart_distribution_trigger = yes }
```

Comment:
`random_character` is CPU heavy. Better replace it by `random_playable_ruler`.

Reply:
Are you sure that you want to exclude baron characters from gaining artifacts on gamestart?

Reply:
why not? barons weren't able to craft artifacts in vanilla either

Reply:
It's NOT about crafting, it's about receiving artifacts on game start

Reply:
Changed to `random_playable_ruler` in df9f3a05